src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -222,13 +222,15 @@ impl KvmVcpu {
     pub fn dump_cpu_config(&self) -> Result<CpuConfiguration, KvmVcpuError> {
         let mut reg_list = get_all_registers_ids(&self.fd).map_err(KvmVcpuError::DumpCpuConfig)?;
 
-        // SYS_CNTV_CVAL_EL0 should be removed, because it depends on the elapsed time and the
-        // dumped CPU config is used to create custom CPU templates to modify CPU features exposed
-        // to guests or ot detect CPU configuration changes caused by firecracker/KVM/BIOS.
+        // SYS_CNTV_CVAL_EL0 and SYS_CNTPCT_EL0 are timer registers and depend on the elapsed time.
+        // This type of registers are not useful as guest CPU config dump.
+        //
         // The value of program counter (PC) is determined by the given kernel image. It should not
         // be overwritten by a custom CPU template and does not need to be tracked in a fingerprint
         // file.
-        reg_list.retain(|&reg_id| reg_id != SYS_CNTV_CVAL_EL0 && reg_id != PC);
+        reg_list.retain(|&reg_id| {
+            reg_id != SYS_CNTV_CVAL_EL0 && reg_id != SYS_CNTPCT_EL0 && reg_id != PC
+        });
```

Comment:
We had a discussion offline and I thought we were aligned to move this filtering to the `cpu-template-helper` tool.

Reply:
I'm still thinking the x86 part yet and will do the migration in a follow-up :)